simplifyDriver.cc

```diff
@@ -82,859 +87,232 @@ using namespace std;
 //! If you wish not to see the output, change the value: verbose = false;
 bool verbose=true;
 
+// Examples
+void CSR_Gauss_Seidel_Example();
+void CSR_ILU_Example();
 
+// Utility function
+bool printRelation(string msg, Relation *rel);
 void EXPECT_EQ(string a, string b);
+void EXPECT_EQ(Relation *a, Relation *b);
+int str2int(string str);
 
-
-int main(int argc, char **argv)
+// Reads information from a JSON file (inputFile), and applies
+// the simplfication algorithm to the sets found in the file. 
+void simplifiy(string inputFile)
 {
-    // (1)
```

Comment:
I replaced back the chill_usage to hand written version without JSON.

On Thu, Jun 23, 2016 at 8:49 PM, Michelle Strout <notifications@github.com>
wrote:

> In simplifyDriver.cc
> <https://github.com/CompOpt4Apps/IEGenLib/pull/38#discussion_r68265793>:
>
> >  {
> > -    // (1)
>
> Where did all of this go? Into another test case? We definitely don't want
> to throw old test cases away even if we have automated them better.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CompOpt4Apps/IEGenLib/pull/38/files/873a894609e6a1eb8305ba539144baac69001790#r68265793>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANGVl6XCeEpAJsJbgIjP_2VtbxXq6E0pks5qOrISgaJpZM4I89jG>
> .
>